### PR TITLE
US106173-update enrollment updates and add demo page

### PR DESCRIPTION
**Pull request by edwardg8**

### CODE REVIEW

File: components/d2l-enrollment-card/d2l-enrollment-updates.js [hunk 1]
```diff
@@ -100,7 +105,7 @@ class EnrollmentUpdates extends mixinBehaviors([
 			'ShowUnreadDropboxSubmissions': this.showUnreadDropboxSubmissions,
 		};
 
-		var notification = this._orgUpdates_fetch(entity, presentationAttributes);
+		var notification = this._orgUpdates_fetch(enrollment.entity(), presentationAttributes);
```
Comment (m6jones) [inline components/d2l-enrollment-card/d2l-enrollment-updates.js]: Does the base class have `entity` now?

Comment (m6jones) [inline components/d2l-enrollment-card/d2l-enrollment-updates.js]: It feels wrong to send entity through like this. Will this be fixed by Cheick?

Comment (m6jones) [reply components/d2l-enrollment-card/d2l-enrollment-updates.js]: Does this need to be organization type?

Comment (edwardg8) [reply components/d2l-enrollment-card/d2l-enrollment-updates.js]: Oh make sense

Comment (svanherk) [reply components/d2l-enrollment-card/d2l-enrollment-updates.js]: Yeah I don't think we should ever need to send the entity back out, is this maybe a `NotificationEntity` or `PresentationEntity` or something?

Comment (m6jones) [reply components/d2l-enrollment-card/d2l-enrollment-updates.js]: Yeah Cheick is working on NotificationCollectionEntity

Comment (edwardg8) [reply components/d2l-enrollment-card/d2l-enrollment-updates.js]: Will be blocked on that for now.

Comment (m6jones) [reply components/d2l-enrollment-card/d2l-enrollment-updates.js]: For the hero banner I am working on. I used this as a stop gap:
```
<d2l-enrollment-updates
	href="[[_notificationsUrl]]"
	token$=[[token]]
	show-unattempted-quizzes
	show-dropbox-unread-feedback
	show-ungraded-quiz-attempts
	show-unread-discussion-messages
	show-unread-dropbox-submissions>
</d2l-enrollment-updates>
.
.
.
_onOrganizationChange(organization) {
	this._organizationName = organization.name();

	// temp waiting on the notifications to get entities.
	this._notificationsUrl = organization._entity && organization._entity.getLinkByRel(Rels.Notifications.organizationNotifications).href;
}
```

**Comment by m6jones** [comment]
Test should be fixed now

**Comment by edwardg8** [comment]
Nice thanks Maya

File: EnrollmentEntity.js [hunk 2]
```diff
@@ -49,6 +49,10 @@ export class EnrollmentEntity extends Entity {
 		return this._entity.getLinkByRel(Rels.Activities.userActivityUsage).href;
 	}
 
+	entity() {
```
Comment (svanherk) [inline EnrollmentEntity.js]: I think we can remove this?

Comment (edwardg8) [reply EnrollmentEntity.js]: OH missed it, you are right, will remove it

**Pull request merged**